package.json

```diff
@@ -4,11 +4,13 @@
   "version": "1.0.0",
   "angular-cli": {},
   "scripts": {
-    "start": "ng serve",
+    "start": "node ./bin/www",
```

Comment:
It is since it breaks the current process. Maybe instead of renaming start to startng, we should have startnode (or something similar). Basically, we'll introduce a new command specifically for node, and leave ng as is. There are a lot of people who cloned this project and if they do a merge their current processes (or many scripts) will break.